qiskit/visualization/latex.py

```diff
@@ -222,236 +222,243 @@ def _get_image_depth(self, aliases=None):
         columns = 2  # wires in the beginning and end
         is_occupied = [False] * self.img_width
         max_column_width = {}
-        for op in self.ops:
-            # useful information for determining row spacing
-            boxed_gates = ['u0', 'u1', 'u2', 'u3', 'x', 'y', 'z', 'h', 's',
-                           'sdg', 't', 'tdg', 'rx', 'ry', 'rz', 'ch', 'cy',
-                           'crz', 'cu3', 'id']
-            target_gates = ['cx', 'ccx']
-            if op.name in boxed_gates:
-                self.has_box = True
-            if op.name in target_gates:
-                self.has_target = True
-
-            # useful information for determining column widths and final image
-            # scaling
-            if op.name not in ['measure', 'reset', 'barrier']:
-                qarglist = op.qargs
-                if aliases is not None:
-                    qarglist = map(lambda x: aliases[x], qarglist)
-                if len(qarglist) == 1:
-                    pos_1 = self.img_regs[(qarglist[0][0],
-                                           qarglist[0][1])]
-                    if op.condition:
-                        mask = self._get_mask(op.condition[0])
-                        cl_reg = self.clbit_list[self._ffs(mask)]
-                        if_reg = cl_reg[0]
-                        pos_2 = self.img_regs[cl_reg]
-                        for i in range(pos_1, pos_2 + self.cregs[if_reg]):
-                            if is_occupied[i] is False:
-                                is_occupied[i] = True
+
+        boxed_gates = ['u0', 'u1', 'u2', 'u3', 'x', 'y', 'z', 'h', 's',
+                       'sdg', 't', 'tdg', 'rx', 'ry', 'rz', 'ch', 'cy',
+                       'crz', 'cu3']
```

Comment:
Why did you remove 'id' from the list?

Reply:
I kept an older version of the list, I have added `'id'` back in.